Cargo.toml

```diff
@@ -22,9 +22,9 @@ members = [
     "blaze",
 ]
 
-[profile.release]
-lto = true
-codegen-units = 1
+#[profile.release]
+#lto = true
+#codegen-units = 1
```

Comment:
curious what's the reason for disabling these optimizations here?

Reply:
For fast `cargo release`, I guess.

Reply:
why not enable these for the release build though? lto has a pretty noticeable impact on performance based on my previous tests